webgpu.yml

```diff
@@ -226,21 +226,34 @@ enums:
         doc: |
           TODO
   - name: callback_mode
-    doc: |
-      TODO
+    doc: The callback mode controls how a callback for an asynchronous operation may be fired. See @ref Asynchronous-Operations for how these are used.
     entries:
       - name: wait_any_only
         value: 0x0001
         doc: |
-          TODO
+          Callbacks created with `WGPUCallbackMode_WaitAnyOnly`:
+          - fire when the asynchronous operation's future is passed to a call to `::wgpuInstanceWaitAny`
+            AND the operation has already completed or it completes inside the call to `::wgpuInstanceWaitAny`.
       - name: allow_process_events
         value: 0x0002
         doc: |
-          TODO
+          Callbacks created with `WGPUCallbackMode_AllowProcessEvents`:
+          - fire when the asynchronous operation's future is passed to a call to `::wgpuInstanceWaitAny`
+            AND the operation has already completed or it completes inside the call to `::wgpuInstanceWaitAny`.
```

Comment:
Instead of repeating, I think I'd slightly prefer if each just refers to the previous mode's allowed behaviors.

Reply:
Done